tests/server/routes/companyRoutes.test.js

```diff
@@ -51,3 +52,21 @@ test('GET /api/v1/companies/id', () => {
       expect(res.body.id).toBe(2)
     })
 })
+
+test ('POST /api/v1/companies/profile/id', () => {
+  const id  = 33
+  return request(server)
+    .post(`/api/v1/companies/profile/${id}`)
+    .send({
+      role_id: 6,
+      company_id: id,
+      count: 3,
+      last_updated_date: "28-03-11",
+      avg_years_exp: 33,
+      female_count: 25,
+      male_count: 15,
+      open_reqs: 32,
+      percent_local: 14
```

Comment:
We shouldn't be using underscores here. These should be camelCase and we should be mapping them to train_case in the db functions.